engine/runtime-compiler/src/main/java/org/enso/compiler/pass/analyse/PassPersistance.java

```diff
@@ -62,10 +64,12 @@
 @Persistable(clazz = AliasMetadata.RootScope.class, id = 1262, allowInlining = false)
 @Persistable(clazz = AliasMetadata.ChildScope.class, id = 1263, allowInlining = false)
 @Persistable(clazz = Graph.Link.class, id = 1266, allowInlining = false)
-@Persistable(clazz = TypeInference.class, id = 1280)
-@Persistable(clazz = FramePointerAnalysis$.class, id = 1281)
-@Persistable(clazz = TailCall.TailPosition.class, id = 1282)
-@Persistable(clazz = CachePreferences.class, id = 1284)
+@Persistable(clazz = TypeInferencePropagation.class, id = 1280)
+@Persistable(clazz = TypeInferenceSignatures.class, id = 1281)
+@Persistable(clazz = FramePointerAnalysis$.class, id = 1282)
+@Persistable(clazz = TailCall.TailPosition.class, id = 1284)
+@Persistable(clazz = CachePreferences.class, id = 1285)
```

Comment:
- @radeusgd, why have you added `CachePreferences` to this list?
- it is causing #11858

Reply:
Right. I was confused. Your PR didn't introduce the persistance declaration.